server/src/main/java/io/unitycatalog/server/utils/Constants.java

```diff
@@ -7,6 +7,7 @@ public class Constants {
   public static final String SCHEMA = "schema";
   public static final String TABLE = "table";
   public static final String FUNCTION = "function";
+  public static final String DEFAULT_METASTORE_NAME = "uc_oss_metastore";
```

Comment:
Wondering if this should be user-settable with some default. The default can contain ID to uniquify across server instances.
@TD Will need your expertise here on deciding the metastore-name.

Reply:
@tdas Will need your expertise here.